src/Logging/HTTPOutputHandler.php

```diff
@@ -70,6 +72,38 @@ public function setStatusCode($statusCode)
     }
 
     /**
+     * Set a formatter to use if Director::is_cli() is true
+     */
+    public function setCliFormatter(FormatterInterface $cliFormatter)
```

Comment:
psr-2 doesn't strictly declare, apparently. https://groups.google.com/forum/#!topic/php-fig/rGopcf4WyyU

I suggest we stick with our own convention until such time as it's clarified.